### DISCUSSION TITLE
Est-il possible de rendre le bandeau obligatoire et de bloquer la navigation ?

### DISCUSSION THREAD

**Discussion by jonathan-ati** [AmauriC/tarteaucitron.js#770] [Q&A]
Bonjour, 

Je viens de me renseigner pour rendre le bandeau des cookies obligatoire. 
j'ai bien ma licence de tarte au citron pour tous les sites web. Mais j'ai remarqué, que l'utilisateur peut ne pas cliquer sur le bandeau et qu'il peut naviguer sur le site et du coup les statistiques ne sont pas activées. Ce qui engendre des baisses sur les statistiques. 

Est-il possible d'obliger à  l'utilisateur de cliquer sur "tout accepter ou tout refuser" pour pouvoir naviguer sur le site ?

Est-ce conforme au RGPD ? 
Merci d'avance pour vos réponses. 

Jonathan

**Comment by AmauriC** [comment]
Bonjour,

En activant le bandeau "centré" depuis la console tarteaucitron le bandeau sera présenté dans un grand popup centré et le taux de réponse sera beaucoup plus important ; )

**Reply by jonathan-ati** [reply]
merci pour votre réponse. Je vais déjà mettre en place cette solution et attendre les retours clients.

**Reply by jonathan-ati** [reply]
Salut, 
Avez-vous aussi rencontré une grande baisse de suivis analytics ? 
Exemple : environ 10000 visiteurs et depuis l'installation de tarte au citron, j'arrive entre 1000 / 1500 ! 

Merci pour vos réponses
A bientôt, 
Jonathan
